BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -169,7 +169,7 @@
                                 {% endif %}
                             </tr>
                             <tr class="accordion-formrow {% if field.vars.valid %}hide{% endif %}{% if field.vars.attr.class is defined %} {{ field.vars.attr.class }}{% endif %}">
-                                <td colspan="{{ prototype.vars.columns|length + 1 + ((form.vars.allow_delete) ? 1 : 0) }}">
+                                <td colspan="{{ prototype.vars.columns|length + 1 + ((form.vars.allow_delete) ? 1 : 0) + ((form.vars.sortable is defined and form.vars.sortable) ? 1 : 0) }}">
```

Comment:
Should be done in prototype too (line 111). That's why I prefer this : https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2183